### DISCUSSION TITLE
Can't delete user account

### DISCUSSION THREAD

**Discussion by ppetermann** [AntlerVC/firetable#1300] [Support / Q&A] [answered]
**Describe the bug**
I tried to delete my account, but I get an error message that I still have an active project in my workspace, however I've left the only workspace I've ever been a part of

**To Reproduce**
1. try to delete account

**Expected behavior**
account is deleted immediately, right to forget is respected

**Comment**
Sorry if this is the wrong place, but trying the support menu didn't get me anywhere.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @ppetermann, I understand that you would like to delete your account. We're here to assist you with that. Could you please provide the following information either here or through [support@rowy.io](mailto:support@rowy.io)?

```
Your Email:
Project ID:
Reason for wanting to delete your account:
```
Thank you!
